srl/src/main/java/edu/illinois/cs/cogcomp/srl/utilities/WeightVectorUtils.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
`illinois-srl` has a dependency to `sl` and `inference` (and `core-utilities` of course). So moving this file to one of those package would be the best solution.